src/models/v1.0/V1WorkflowModel.ts

```diff
@@ -218,7 +228,10 @@ export class V1WorkflowModel extends WorkflowModel implements Serializable<Workf
 
         base.requirements = requirements;
 
-        delete this.customProps.requirements;
+        /* Deleting all user supplied requirements is inadvisable
+           It is best to superimpose detected requirements on top of any
+           the User has supplied */ 
+        //delete this.customProps.requirements;
```

Comment:
Yes, this causes the following problem:

1. Take a workflow that has a requirement that is not on the list to be added (and which our feature detector does not deal with)
2. Run it once on the test pane. It executes fine.
3. Run it again. It fails. This can be tracked down to any requirement not on the list we check being deleted. The requirement is only added back into `customeProps.requirements` when the model is regenerated from the user's code.

Reply:
Understandable, then we can delete this whole line with comment, and move this comment to commit. Then we are all set for a release 👍